testing/e2e/suites/utils.go

```diff
@@ -28,7 +28,7 @@ func waitForAvailable(ctx context.Context, c client.Client, deployment appsv1.De
 			}
 		}
 
-		if time.Since(start) > 10*time.Minute { // 10 minutes because it takes a decent amount of time for dns to "propagate"
+		if time.Since(start) > 20*time.Minute { // 10 minutes because it takes a decent amount of time for dns to "propagate"
```

Comment:
why does this need to be bumped to 20 minutes?

Reply:
the local run I showed you succeeded, it just needed more time - believe I made this change on a call w you

Reply:
how much time does it need and why?

i suggested the change to see if it works but we need to figure out why. it's bad if it takes so long for things to start working after just a pod restart.

Reply:
Per [this form](https://learn.microsoft.com/en-us/answers/questions/524792/azure-rbac-propagation-latency) rbac propagation can take up to 30 minutes - we can add timeout as an argument to have different timeouts between when the dns contributor role is first assigned (the first testing iteration) and when nginx is restarted, but that seems like overkill